spec/components/admin/npq/applications/analysis/table_row_spec.rb

```diff
@@ -1,38 +1,26 @@
 # frozen_string_literal: true
 
-RSpec.describe Admin::NPQ::Applications::Analysis::TableRow, type: :view_component do
-  let!(:cohort) { create :cohort }
-  let!(:schedule) { create :npq_leadership_schedule, cohort: }
-  let(:npq_course) { create :npq_course, identifier: "npq-senior-leadership" }
-  let(:npq_lead_provider) { create :npq_lead_provider }
-
-  let(:application) do
-    npq_application = create(:npq_application, :rejected,
-                             npq_lead_provider:,
-                             npq_course:,
-                             cohort:)
-
-    participant_profile = create(:npq_participant_profile, npq_application:)
+RSpec.describe Admin::NPQ::Applications::Analysis::TableRow, :with_default_schedules, type: :view_component do
+  let(:npq_lead_provider) { create(:cpd_lead_provider, :with_npq_lead_provider).npq_lead_provider }
+  let(:npq_application) do
+    create(:npq_application, :funded, :accepted, npq_lead_provider:)
+  end
 
+  before do
     create(:npq_participant_declaration,
-           declaration_type: "started",
-           user: npq_application.user,
-           participant_profile:,
-           cpd_lead_provider: npq_lead_provider.cpd_lead_provider,
-           course_identifier: npq_course.identifier,
-           state: "paid")
-
-    npq_application
+           :paid,
+           participant_profile: npq_application.profile,
+           cpd_lead_provider: npq_lead_provider.cpd_lead_provider)
   end
 
-  component { described_class.new application: }
+  component { described_class.new application: npq_application }
 
-  it { is_expected.to have_content "Full name\n    #{application.user.full_name}" }
-  it { is_expected.to have_link application.user.full_name, href: admin_participant_path(application.profile) }
-  it { is_expected.to have_content "Application Status\n    rejected" }
-  it { is_expected.to have_content npq_course.name }
+  it { is_expected.to have_content "Full name\n    #{npq_application.user.full_name}" }
```

Comment:
I know this wasn't added in this PR but here we can make it more readable using [normalize_ws](https://www.rubydoc.info/gems/capybara/Capybara%2FNode%2FMatchers:assert_text):

```
it { is_expected.to have_css(
  ".nhsuk-table__cell",
  text: "Full name #{npq_application.user.full_name}",
  normalize_ws: true
) }
```